pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/TableConfigsRestletResource.java

```diff
@@ -304,8 +304,10 @@ public ConfigSuccessResponse updateConfig(
       @ApiParam(value = "comma separated list of validation type(s) to skip. supported types: (ALL|TASK|UPSERT)")
       @QueryParam("validationTypesToSkip") @Nullable String typesToSkip,
       @ApiParam(value = "Reload the table if the new schema is backward compatible") @DefaultValue("false")
-      @QueryParam("reload") boolean reload, String tableConfigsStr)
-      throws Exception {
+      @QueryParam("reload") boolean reload,
+      @ApiParam(value = "Force update the table schema") @DefaultValue("false")
```

Comment:
Could we name this as forceTableSchemaUpdate just to be clear about what we are forcing? Also could we  use this name in the associated functions.

Reply:
+1 on calling it forceSchemaUpdate